applications/IgaApplication/tests/cpp_tests/test_support_laplacian_condition.cpp

```diff
@@ -4,96 +4,86 @@
 //   _|\_\_|  \__,_|\__|\___/ ____/
 //                   Multi-Physics
 //
-//  License: BSD License
-//           Kratos default license: kratos/license.txt
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Nicolò Antonelli
 //
 
 // Project includes
 #include "containers/model.h"
 #include "testing/testing.h"
-
 #include "test_creation_utility.h"
-
 #include "custom_conditions/support_laplacian_condition.h"
 #include "includes/convection_diffusion_settings.h"
 
-namespace Kratos
+namespace Kratos::Testing
 {
-namespace Testing
-{
-    ///@name Type Definitions
-    ///@{
-
-    typedef std::size_t SizeType;
-    typedef std::size_t IndexType;
-
-    ///@}
-    ///@name Operations
-    ///@{
-
-    typename Condition::Pointer GetSupportLaplacianCondition(
-        ModelPart& rModelPart, SizeType PolynomialDegree, IntegrationPoint<3> IntegrationPoint)
+    namespace
```

Comment:
You're including everything in the anonymous namespace! Only the auxiliary functions should be there...